static/app/components/events/groupingInfo/groupingInfo.tsx

```diff
@@ -81,12 +84,29 @@ export default function GroupingInfo({
           </div>
```

Comment:
unrelated to this PR specifically but when you open the fold section for the first time the feedback button loads outside the fold section
<img width="1212" height="89" alt="Screenshot 2025-08-12 at 3 06 59 PM" src="https://github.com/user-attachments/assets/28e28f24-bdae-4c0b-a42f-2dd5eb674eab" />